test/convertToPrimitive.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
+const { Big } = require('../lib/bigdecimal.js');
+const chai = require('chai');
+chai.should();
+
+/* eslint-disable eqeqeq */
+describe('Convert to primitive test', function () {
+    it('should throw when calling JSON.stringify on BigDecimal', () => {
+        const testCase = () => JSON.stringify(Big(0));
+        testCase.should.throw('Do not know how to serialize a BigInt', undefined, 'expected JSON.stringify to throw');
+    });
+
+    it('should be able to convert to string (hint: string)', () => {
+        const value = Big(0);
+        const testCases = {
+            '`${Big(0)}`': [() => `${value}`, '0'],
+        };
+
+        Object.entries(testCases).forEach(([key, [testCase, expected]]) => {
+            testCase().should.equals(expected, `expected '${key}' to be '${expected}'`);
+        });
+    });
+
+    it('should perform logical operators on BigDecimal without calling toPrimitive', () => {
```

Comment:
if toPrimitive is not used with these, do we need this test? I mean, we are testing the Javascript's behaviour I think.